pkg/geoip/geoip.go

```diff
@@ -47,7 +52,11 @@ func getLocation(geoIPService string) (Location, error) {
 		City:      "Unknown",
 	}
 
-	resp, err := http.Get(geoIPService)
+	cl := &http.Client{
```

Comment:
the geoip pkg is kinda different. Although i don't like global vars may be instead of declaring the timeout as const we can have a global (private) defaultClient in this pkg this way we also don't have to create an instance everytime. and will behave similar to `http.DefaultClient`